### ISSUE TITLE
Colony seems to be loaded when player is far away with forceloadcolony enabled

### ISSUE THREAD

**Issue by Titanium237458** [ldtteam/minecolonies#6749]
### Prerequisites 

- [x] I am running the latest **alpha** version of MineColonies and Structurize for my Minecraft version.
- [x] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure my issue is not covered there.
- [x] I made sure that **this issue is not a duplicate of any existing issue.**

### Context 

- Minecraft Version: 1.16.5
- MineColonies Version: 0.14.19
- Structurize Version: 0.13.148
- Related Mods and their Versions: None

## Expected behavior

Colony is only loaded when player is nearby.

## Actual behavior

It seems that claimed chunks are almost always chunkloaded even when the player is around 10000 blocks away when the forceloadcolony config is enabled. 

![screenshot](https://user-images.githubusercontent.com/8401588/111342728-0650f380-867b-11eb-9fb3-77f93c756da9.jpg)

Notice in the screenshot how my current position is around 0 0 and I got a message from my builder who finished a build around 10000 10000.

## Steps to reproduce the issue

1. Create a new world (creative, preferably Superflat with Redstone Ready Preset)
2. Turn on forceloadcolony in the serverconfig of that world.
3. Set colonyloadstrictness in config to 1 (optional, but I was able to encounter the issue more reliably)
4. Create a colony with townhall, builder and a guardtower (to increase the amount of claimed chunks)
5. Put a new unfinished building in one of the claimed chunks and make sure the whole building is in said chunk (i.e. cave housing level 1)
6. Increase builder's stats for them to build faster (optional) and give them all the needed tools and materials.
7. Wait for the builder to enter the same chunk as the construction site (optional if colonyloadstrictness is set to 1, but builder needs to be in claimed chunk).
8. Teleport far away and wait for the builder to finish the build.

### Logs

- latest.log:  No AI issue
- crashlog:  No crash

### Notes

I believe this is a bug since the description in the config file implies that the colony is only loaded when part of it is loaded by a player.
---
## Viewers

* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by someaddons** [comment]
that is exactly the function of that config to keep colony chunks loaded when the player is away. When you do not want that do not enable the config

**Comment by Raycoms** [comment]
They will unload only after being far away for a considerable long time

**Comment by Titanium237458** [comment]
It's still a useful option to have, if I'm at one end of a big colony and the other end is still loaded, but I'd rather not have barbarians spawn when I'm out exploring far away.
And how long exactly is "a considerable long time", if I may ask?

**Comment by someaddons** [comment]
about 10 minutes
